.babelrc

```diff
@@ -35,6 +35,12 @@
       "plugins": [
         "./build/use-lodash-es"
       ]
+    },
+    "rollup": {
+      "plugins": [
+        "./build/use-lodash-es",
+        "external-helpers"
```

Comment:
OK, I apparently have a fundamental misunderstanding of babel-plugin-external-helpers or rollup-plugin-babel does :P

external-helpers is supposed to tell the transpiled code to look at `global` for the `babelHelpers` property to find them. That means they should be external (like the name implies) from the build and included via a separate `<script>` tag. For whatever reason, rollup-plugin-babel actually treats them like an inline-able dependency, which is what transform-runtime is supposed to do. 

Anyways, that's just my 2 cents. Since the naming is backwards, this is actually what we want and I was wrong about it originally :)